bot.js

```diff
@@ -1,114 +1,59 @@
-`use strict`;
+"use strict";
 
 const { ChatClient, AlternateMessageModifier } = require(`dank-twitch-irc`);
 const config = require(`./config`);
 const client = new ChatClient(config.opts);
 const talkedRecently = new Set();
-const pajladaID = `11148817`;
+const pajladaID = "11148817";
 
-function isUpperCase(str) {
-  return str === str.toUpperCase();
-};
-
-function isLowerCase(str) {
-  return str === str.toLowerCase();
-};
+const isUpperCase = (str) => { return (str === str.toUpperCase()) };
```

Comment:
For someone who is fairly noobish at coding, I think "condensing" code just to preserve lines makes the readability far worse for myself.